docs/en/setup/backend/backend-pulsar-monitoring.md

```diff
@@ -0,0 +1,67 @@
+
+## Pulsar Monitoring
+
+Pulsar monitoring provides multidimensional metrics monitoring of Pulsar cluster as `Layer: PULSAR` `Service` in
+the OAP. In each cluster, the nodes are represented as `Instance`.
+
+### Pulsar Cluster Supported Metrics
+
+| Monitoring Panel     | Metric Name                                | Description                                                                                            | Data Source    |
+|----------------------|--------------------------------------------|--------------------------------------------------------------------------------------------------------|----------------|
+| Total Topics         | meter_pulsar_total_topics                  | The number of Pulsar topics in this cluster.                                                           | Pulsar Cluster |
+| Total Subscriptions  | meter_pulsar_total_subscriptions           | The number of Pulsar subscriptions in this cluster.                                                    | Pulsar Cluster |
+| Total Producers      | meter_pulsar_total_producers               | The number of active producers connected to this cluster.                                              | Pulsar Cluster |
+| Total Consumers      | meter_pulsar_total_consumers               | The number of active consumers connected to this cluster.                                              | Pulsar Cluster |
+| Message Rate In      | meter_pulsar_message_rate_in               | The total message rate coming into this cluster (message per second).                                  | Pulsar Cluster |
+| Message Rate Out     | meter_pulsar_message_rate_out              | The total message rate going out from this cluster (message per second).                               | Pulsar Cluster |
+| Throughput In        | meter_pulsar_throughput_in                 | The total throughput coming into this cluster (byte per second).                                       | Pulsar Cluster |
+| Throughput Out       | meter_pulsar_throughput_out                | The total throughput going out from this cluster (byte per second).                                    | Pulsar Cluster |
+| Storage Size         | meter_pulsar_storage_size                  | The total storage size of all topics in this broker (in bytes).                                        | Pulsar Cluster |
+| Storage Logical Size | meter_pulsar_storage_logical_size          | The storage size of all topics in this broker without replicas (in bytes).                             | Pulsar Cluster |
+| Storage Write Rate   | meter_pulsar_storage_write_rate            | The total message batches (entries) written to the storage for this broker (message batch per second). | Pulsar Cluster |
+| Storage Read Rate    | meter_pulsar_storage_read_rate             | The total message batches (entries) read from the storage for this broker (message batch per second).  | Pulsar Cluster |
```

Comment:
The units such as `(message per second)`  you add to the description should be added to the dashboards too.